### ISSUE TITLE
Does not fully work to comtrol pdf viewing actions in Firefox

### ISSUE THREAD

**Issue by Noexpert** [philc/vimium#2843]
When viewing a pdf in Firefox (57), the keys to zoom in, zoom out, select a zoom level, change page no, print, download all work.

However the key to switch to full screen "Presentation Mode" does not work.

**Comment by LangdalP** [comment]
I have a similar problem with using the "f" bind to open video players on twitch and youtube in full screen. In the developer console, I get the following error:
```
Request for full-screen was denied because Element.mozRequestFullScreen() was not called from inside a short running user-generated event handler.
```
I just tried to do what you described above, and I see that this also results in the error shown above.

**Comment by nimafazeli** [comment]
The plugin used to work on built-in pdf viewer of firefox but it has been a few days that I noticed it does not work anymore. I am using firefox 60. 

On another note it does work on the *Reader view* either. 
It is understandable that plugins are disabled for settings or add-on interfaces but I do not get it why Firefox disables them on pdf or reader view.

**Comment by sxmwht** [comment]
I'll preface by saying I am fairly inexperienced with github etiquette, so I'm not sure if commenting on such an old issue is considered poor form, so please forgive me. 

Firefox PDF viewer in its current state rejects all Vimium commands, which is very jarring. I want to be able to scroll (smoothly) with j and k, and close with x. Even if this small subset of commands were available, it'd be a much better UX. 

If this is a Firefox limitation then my apologies.

**Comment by gdh1995** [comment]
yes it's a limitation of Firefox (and also Chrome). You may try my `Vimium C` and `PDF Viewer for Vimium C` if you use Chrome.

**Comment by philc** [comment]
Closing; not much we can do, unfortunately.
